Libraries/LibURL/Pattern/ConstructorStringParser.cpp

```diff
@@ -0,0 +1,706 @@
+/*
+ * Copyright (c) 2025, Shannon Booth <shannon@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#define URL_PATTERN_DEBUG 1
```

Comment:
This appears to be left over from debugging.

Reply:
whoopsy, thanks for all; forgot about all the dbgln_if stuff